index.js

```diff
@@ -115,7 +115,7 @@ module.exports = function(options) {
       data = file.contents.toString('utf8');
       data = data.replace(/'/g, '"');
       data = data.replace(/\s+/g, " ");
-      data = data.replace(/[\(\){}\|\\\^~\[\]`"<>#%]/g, function(match) {
+      data = data.replace(/[\(\){}\|\\\^~\[\]`"\'<>#%]/g, function(match) {
```

Comment:
Hey @FlyingDR I oped another PR #5 which hold the similar fix, however you do not need to encode `\'` . with regex because line https://github.com/npostulart/gulp-sass-image/pull/3/files#diff-168726dbe96b3ce427e7fedce31bb0bcR116 already replacing all the `'` with `"`.

As an additional you would want with double quotes for `background-image: url("")`, according to  https://developer.mozilla.org/en-US/docs/Web/CSS/background the is issue is mussing quotes for 

I put also issue #3 as a explanatory reference.